pypeit/spectrographs/mmt_bluechannel.py

```diff
@@ -178,18 +193,20 @@ def default_pypeit_par(cls):
         # 1D wavelength solution
         par['calibrations']['wavelengths']['rms_threshold'] = 0.5
         par['calibrations']['wavelengths']['sigdetect'] = 5.
-        par['calibrations']['wavelengths']['fwhm']= 5.0
-        # HeNeAr is by far most commonly used, though ThAr is used for some situations.
-        par['calibrations']['wavelengths']['lamps'] = ['ArI', 'ArII', 'HeI', 'NeI']
+        par['calibrations']['wavelengths']['fwhm_fromlines'] = True
+
+        # Parse the lamps used from the image header.
+        par['calibrations']['wavelengths']['lamps'] = ['use_header']
```

Comment:
So currently this says below that the wavelength calibration is done with holy-grail. You overwrite this below in config_specific_par, but I'm wondering what the most transparent approach is here. In other words, do you want to default to holy-grail if config_specific_par does not find a match? I think this does that, and one can debate about how to make that most explicit. But perhaps it would be more transparent to the user if an exception was raised in config_specific_par if the grating setup is not calibrated. Currently it will just run with holy-grail in that case which may or may not work, and if it fails it will throw a cryptic exception.

Reply:
The desired behavior here sort of depends on how likely such a corner case is to occur.

Reply:
yes, the intention here is to default to `holy-grail` and override in `config_specific_par` if there is an available template. this is how `mmt_binospec` works currently and until this PR gets merged. in reality, the blue channel templates here cover all cases i've found in the archived data going back to 2014. the only missing dispersers are an old 600 l/mm grating and the echellette mode. the former has been replaced by the 832 l/mm grating which is vastly superior in every way. the latter hasn't been used in over 20 years and i don't think was ever re-commissioned after changing the detector format. if for some weird reason someone did choose to use the 600 l/mm grating (and assuming we can find where it's stored), there is reasonable expectation that `holy-grail` would work. it works pretty well out of the box with the 800 and 500 gratings at similar central wavelength settings.